crates/bevy_transform/src/systems.rs

```diff
@@ -19,6 +20,21 @@ pub fn sync_simple_transforms(
         *global_transform = GlobalTransform::from(*transform);
     });
 }
+/// Update [`GlobalTransform`] component of entities that aren't in the hierarchy
+/// and lost their parent last frame.
+///
+/// Third party plugins should use [`transform_propagate_system_set`](crate::transform_propagate_system_set)
+/// to propagate transforms correctly.
+pub fn sync_simple_transforms_orphaned(
```

Comment:
Could this just be incorporated into sync_simple_transforms? Thus cannot parallelize with it anyway due to the queries not being disjoint.

Reply:
Specifically because they are not disjoint it makes it difficult to merge the two. I could use a `ParamSet`, but then I felt like a distinct system was better.

If there was a `NotChanged<Foo>` filter, it could allow running the two systems in parallel though.

Reply:
@james7132 would you find a `ParamSet` in a single system an acceptable stop gap? I feel this PR shouldn't be blocked on the hypothetical addition of `NotChanged`.

Reply:
As of #7097, you could do

```rs
pub fn sync_simple_transforms_orphaned(
    mut query: Query<(Ref<Transform>, &mut GlobalTransform), Without<Children>>,
    orphaned: RemovedComponents<Parent>,
) {
    let mut iter = query.iter_many_mut(orphaned.iter());
    while let Some((transform, mut global_transform)) = iter.fetch_next() {
        if transform.is_changed() {
            continue;
        }
        *global_transform = GlobalTransform::from(*transform);
    }
}
```